packages/mobile_app/lib/screens/settings/settings_screen.dart

```diff
@@ -0,0 +1,200 @@
+import 'index.dart';
+
+part 'prophecies_enabling.dart';
+
+class SettingsScreen extends StatelessWidget {
+  const SettingsScreen({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final user = AppGlobal.data.usersRepo.current.model;
+    final name = MutableString(user.name);
+    final birthDate = DateTime.fromMillisecondsSinceEpoch(user.birth);
+    final month = MutableString(birthDate.month.toString());
+    if (month.wrapped.length == 1) month.wrapped = "0${month.wrapped}";
+    final day = MutableString(birthDate.day.toString());
+    final year = MutableString(birthDate.year.toString());
+    final sex = MutableInteger(user.sex);
+    final indexToSex = {
+      0: localeText.notSelectedSex.capitalize(),
+      1: localeText.male.capitalize(),
+      2: localeText.female.capitalize(),
+      3: localeText.other.capitalize(),
+    };
+    final MutableBool luck =
```

Comment:
why are you using `MutableBool` here? this value seem to be unchanged

Reply:
MutableWrappers is an old overkill for our userinfo widgets that are used in registration and settings.
I don't want to refactor this part.
But also, this screen is not working correctly because I forget to make it stateful, since it needs an initial state.

Reply:
Yep, I guess the fix comes in #108 

But its still `MutableBool`. Well, I might have a knowledge gap here